rules/cloud/aws/aws_enum_logging.yml

```diff
@@ -0,0 +1,31 @@
+title: Enumerate Backup Configuration on AWS
+id: 76255e09-755e-4675-8b6b-dbce9842cd2a
+status: experimental
+description: Identifies enumeration activity targeting the AWS backups
+references:
+    - https://unit42.paloaltonetworks.com/compromised-cloud-compute-credentials/
+author: Janantha Marasinghe
+date: 2022/12/13
+tags:
+    - attack.discovery
+    - attack.t1580
+logsource:
+    product: aws
+    service: cloudtrail
+detection:
+    selection:
+        eventSource: ec2.amazonaws.com
+        eventName: 
+            - GetPasswordData
+            - GetEbsEncryptionByDefault
+            - GetEbsDefaultKmsKeyId
+            - GetBucketReplication
+            - DescribeVolumes
+            - DescribeVolumesModifications
+            - DescribeSnapshotAttribute
+            - DescribeSnapshotTierStatus
+            - DescribeImages
```

Comment:
I think these need to occur in a certain timeframe for it to be suspicious enumeration. As individual events, these could occur during normal operation

Reply:
> I think these need to occur in a certain timeframe for it to be suspicious enumeration. As individual events, these could occur during normal operation

fixed